db/schema.rb

```diff
@@ -34,6 +34,7 @@
     t.integer "event_id", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.text "edition_diff"
```

Comment:
Another one I suggested. 

If the data is stored as JSON you lose the type of objects when converting. Eg if you have a Date object in the diff, a JSON representation would only tell you it's a string timestamp. Whereas text serialised stores the diff accurately.